### ISSUE TITLE
[Android] Camera Image is rotated 90°

### ISSUE THREAD

**Issue by ohmyguigs** [ideacreation/react-native-barcodescanner#54]
I'm using React-Native 0.22.2 and Android SDK with API level 19 (KitKat).
Camera shows up, read barcodes with success, but image is displayed 90° clockwise.
viewFinder and viewFinderDrawLaser shows up on the right position (not rotated).
I have the permissions on my AndroidManifest.xml:
`<uses-permission android:name="android.permission.CAMERA" />`
App only works in portrait mode, so phone is locked in portrait and orientetion never changes.
This is what my component renders:
`  render() {
    let camera = <View />;
    let childrenContainerStyle = {
      flex: 1
    };

```
if(this.props.status == 'in-use') {
  camera = (
    <CameraScanner
      onBarCodeRead={this.onBarCodeRead}
      style={styles.barcodeScanner}
      torchMode={'off'}
      viewFinderDrawLaser={true}
      cameraType={'back'}
    />
  );
}

return (
  <View style={[this.props.style]}>
    <View style={childrenContainerStyle}>
      {this.props.children}
    </View>
    {camera}
  </View>
);
```

  }
}

var styles = StyleSheet.create({
  barcodeScanner: {
    flex: 1
  }`

tl;dr -> it's all working good, but camera image is flipped 90° clockwise. 😢

**Comment by grundmanise** [comment]
Same here. Camera rotated 90°. (tested only in simulator, using notebook webcamera)

**Comment by ohmyguigs** [comment]
@grundmanise if you need so, im using like this 
https://github.com/resendefreitas/react-native-barcodescanner
😁  for now 😅
